src/components/page_components/study_plans/StudyPlanTable.tsx

```diff
@@ -10,6 +10,7 @@ interface StudyPlanTableProps {
 	elements: StudyPlanElement[];
 }
 
+// TODO: Make the env files .env.local and .env.production.local or w/e
```

Comment:
Agreed with this one, and let's try to have the `API_BASE_LINK` declared globally or smt just so we can simply reuse it over and over